EIPS/eip-3041.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 3041
+title: Add basefee in eth_getBlockByHash response
+author: Abdelhamid Bakhta (@abdelhamidbakhta)
+discussions-to: https://ethereum-magicians.org/t/eip-3041-add-basefee-in-eth-getblockbyhash-response/4825
+status: Draft
+type: Standards Track
+category : Interface
+created: 2020-10-13
+requires: 1559
+---
+
+## Simple Summary
+Add basefee field in `eth_getBlockByHash` RPC endpoint response.
+
+## Abstract
+[EIP-1559](/EIPS/eip-1559) introduces a base fee per gas in protocol.
+This value is maintained under consensus as a new field in the block header structure.
+This change should be reflected in RPC methods that return block header information.
+
+## Motivation
+Users may need value of the base fee at a given block. Base fee value is important to make gas price predictions more accurate.
```

Comment:
```suggestion
[EIP-1559](/EIPS/eip-1559) introduces a base fee per gas in protocol.
This value is maintained under consensus as a new field in the block header structure.
Users may need value of the base fee at a given block. Base fee value is important to make gas price predictions more accurate.
```
Consider moving these sentences from the Abstract to here.  I think they are valuable, just not Abstract material is all.